tardis/model/tests/test_base.py

```diff
@@ -442,8 +415,12 @@ def non_uniform_model_state(atomic_dataset, example_model_file_dir):
         ((28, 1), 9.54725917e-23),
     ],
 )
-def test_radial_1d_model_atomic_mass(non_uniform_model_state, index, expected):
-    atomic_mass = non_uniform_model_state.composition.atomic_mass
+def test_radial_1d_model_atomic_mass(
```

Comment:
Rename to "simulation_state_atomic_mass.

Reply:
done and added some more